### PR TITLE
Allow Espejo to sync OpenShift EgressNetworkPolicy objects by default

### PR DESCRIPTION
**Pull request by megian**
## Checklist

- [x] Keep pull requests small so they can be easily reviewed.
- [ ] Update the documentation.
- [ ] Update the ./CHANGELOG.md.
- [ ] Link this PR to related issues.

### CODE REVIEW

File: class/defaults.yml [hunk 1]
```diff
@@ -45,3 +45,7 @@ parameters:
       - apiGroups: [extensions]
         resources: [networkpolicies]
         verbs: ['*']
+      # OpenShift specific
+      - apiGroups: [network.openshift.io]
+        resources: [egressnetworkpolicies]
+        verbs: ['*']
```
Comment (ccremer) [inline class/defaults.yml:51]: I'm not sure if this is the correct way to add distribution-specific items here. Should we rather do this via parameters in hierarchy?
@corvus-ch  or @srueg WDYT?

Comment (srueg) [reply class/defaults.yml:52]: Yes, I'd suggest you add this in the hierarchy: https://git.vshn.net/syn/commodore-defaults/-/blob/master/distribution/openshift.yml

Comment (srueg) [reply class/defaults.yml:52]: Adding the following should do the trick (it gets merged with the rest of the array):
```
parameters:
  espejo:
    cluster_role_rules:
      - apiGroups: [network.openshift.io]
        resources: [egressnetworkpolicies]
        verbs: ['*']
```

Comment (corvus-ch) [reply class/defaults.yml:52]: I agree with @srueg. Unless a component is clearly targeting a specific distribution (like the once having OpenShift in their name), nothing distribution specific should be part of a component.

Comment (megian) [reply class/defaults.yml:52]: Works as described and a much cleaner solution. Thanks. Closing #14.

**Pull request closed without merging**